api/HTMLMediaElement.json

```diff
@@ -1647,23 +1647,35 @@
             "ie": {
               "version_added": "9"
             },
-            "opera": {
-              "version_added": true
-            },
-            "opera_android": {
-              "version_added": null
-            },
+            "opera": [
```

Comment:
This should be simplified if the right version for Chrome is found. No gap in support here.